selectionsort.py

```diff
@@ -0,0 +1,10 @@
+A = [2, 23, 19, 57, 12]

```

Comment:
instead of hard coding values, use user-inputs.
As program comments, you may add values to show an example